framework/include/reporters/ConstantReporter.h

```diff
@@ -49,6 +52,12 @@ ConstantReporter::addReporterTypeParams(const std::string & prefix, bool add_vec
         prefix + "_vector_names", "Names for each vector of " + prefix + "s value.");
     params.addParam<std::vector<std::vector<T>>>(prefix + "_vector_values",
                                                  "Values for vectors of " + prefix + "s.");
+
+    params.addParam<std::vector<ReporterValueName>>(prefix + "_vector_vector_names",
+                                                    "Names for each vector of vectors of " +
+                                                        prefix + "s value.");
+    params.addParam<std::vector<std::vector<std::vector<T>>>>(
```

Comment:
Why a vector of vectorOfVectors?  Is this so users can input several vectorOfVectors in the same ConstantReporter?   I didn't see a test for multiple vectorOfVectors in the same ConstantReproter, I'm not sure what the input file format would be for it.  I would know how to enter a single vectorOfVectors which is what you have in your transfer test.  Do you think you need to enable a vector of these vectorOfVector reporters?  Maybe just a single one is fine although its different from the ConstantReporters.

Reply:
This is the only hangup, I can see to get this PR merged. Technically a users can never supply more than 1 vector of vector T reporter values, so if they have 2 of those reporter names it would error out. I don't see an easy way to fix this problem, so would it be better to just use a one of the "test" reporters that we have?

Reply:
there is a vector of vector of vector input syntax. with ; and | separators